tests/test_pagination.py

```diff
@@ -99,6 +100,20 @@ def pagination_test(self):
                     location_id = [location['id'] for location in self.client.get_all('locations')][0]
                     for i in range(num_to_post):
                         new_objects.append(self.client.create_order(location_id))
+                elif stream == 'shifts':
+                    # Find the max end_at to know when the last shift ends, so we can start a shift there
+                    max_end_at = max([obj['end_at'] for obj in existing_objects])
+                    end_at_datetime = utils.strptime_to_utc(max_end_at)
+                    for i in range(num_to_post):
+                        # Tested in the API Explorer that +00:00 works
+                        # How does this work? It feels like it should be new_objects += blah
```

Comment:
What is this comment asking? `+=` concatenates two lists, `append` appends to the end of the list.

Reply:
Will remove comment in subsequent PR